openstack/examples/idr-example-omero-bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
+# gcc needed for ansible dependencies
+yum install -q -y git python-virtualenv gcc 2>&1
+virtualenv -q --system-site-packages /opt/ansible 2>&1
+/opt/ansible/bin/pip -q install ansible 2>&1
+
+cd /opt
+
+git clone -q --single-branch -b omego https://github.com/manics/infrastructure.git /opt/infrastructure 2>&1
+
+# Custom variable overrides (YAML, can be empty)
+cat << EOF > /opt/infrastructure/localhost-extravars.yml
+omero_release: "5.2.2"
+#omero_release: OMERO-build
```

Comment:
I've had to add it to the extra_vars line rather than the extravars.yml. Unsure why.